bpython/curtsiesfrontend/repl.py

```diff
@@ -844,6 +846,13 @@ def on_control_d(self):
             self.current_line = (self.current_line[:self.cursor_offset] +
                                  self.current_line[(self.cursor_offset + 1):])
 
+    def on_control_o(self):
```

Comment:
We probably want an escape hatch at the beginning of this method like
~~~
if self.rl_history.index == 0:
    self.on_enter()
    return
~~~
so ctrl-o works like enter if we aren't back in history.